fractal_server/main.py

```diff
@@ -81,6 +82,7 @@ def check_settings() -> None:
 
 @asynccontextmanager
 async def lifespan(app: FastAPI):
+    app.state.jobs = []
```

Comment:
If we handle both V1 and V2 jobs, then need two attributes (`jobs_v1` and `jobs_v2`).